### PR TITLE
Cauchy|Added Info component as well as mmr instruction on MMR page.

### PR DESCRIPTION
**Pull request by CauchyAbel**
image

UI as shown, new to react xD

### CODE REVIEW

**Comment by CauchyAbel** [comment]
@blukai @howardchung Fix for #1121 , please review, thanks!

File: src/lang/en-US.json [hunk 1]
```diff
@@ -443,6 +443,8 @@
   "matches_highest_mmr": "High MMR",
   "matches_lowest_mmr": "Low MMR",
 
+  "mmr_not_up_to_date": "Why is my mmr not up to date?",
```
Comment (howardchung) [inline src/lang/en-US.json]: capitalize MMR

**Comment by CauchyAbel** [comment]
image

Change pushed, please review @blukai @howardchung

**Comment by blukai** [comment]
Is it possible to make it show if last mmr update was more than ~1 day ago for example?

**Comment by CauchyAbel** [comment]
@blukai I should be able to implement that, will give it a try :) 👍

**Comment by CauchyAbel** [comment]
@blukai Just started up the front end and found that if you don't have a game a day ago, the MMR graph won't show MMR for that day, seems it is only catching the MMR changes. For example I haven't played ranked game for 5 days and it only shows my MMR until 5 days ago. Don't know how we could detect the MMR update though.

**Comment by blukai** [comment]
```js
{
  account_id: 210377205,
  match_id: 3141276771,
  solo_competitive_rank: 5481,
  competitive_rank: null,
  time: "2017-04-26T15:09:11.489Z"
}
```
I guess u can compare the last time when it was updated to current time

**Comment by CauchyAbel** [comment]
@blukai But I mean it is still possible that MMR was updated 5 (more than 1 day) days ago even though it is up to date. Just like in my case I didn't play for five days so the updated time should have 5 days difference to current date.

**Comment by howardchung** [comment]
I think it's fine to just always show the message.

**Pull request merged**